ethereum/test/test_eth_app.js

```diff
@@ -147,23 +150,24 @@ contract("ETHApp", function (accounts) {
       const recipient = "0xcCb3C82493AC988CEBE552779E7195A3a9DC651f";
 
       // expected amount to unlock
-      const amount = BigNumber(web3.utils.toWei("1", "ether"));
+      const amount = web3.utils.toWei("1", "ether");
 
       const beforeBalance = BigNumber(await this.app.balance());
       const beforeRecipientBalance = BigNumber(await web3.eth.getBalance(recipient));
 
       // Commitment payload generated using:
       //   cd parachain/pallets/eth-app
       //   cargo test test_outbound_payload_encode -- --nocapture
-      const commitment = [
+      const messages = [
```

Comment:
Strictly speaking, `messages` is the commitment, so I would wouldnt rename this. The `commitment` variable you added is better referred to as the `commitmentHash`

Reply:
i think its cleaner to leave it as messages - i know the data is all being generated by the commitment pallet, but the commitment generated by that pallet is just the hash, not the messages?

Reply:
The pallet generates two things, a commitment (list of messages), and a hash of the former. That's the terminology used across the system (parachain, relayer, solidity) since I landed the Big Channels PR. The relayer passes a `commitment` and `commitmentHash` to the contract. With that in mind, I think its important to be consistent about naming across our system, as it leaves less room for confusion.

Reply:
I think `messages`/`commitment` or `messages`/`commitmentHash` makes more sense but I agree we should stick to the existing terminology.

Reply:
@vgeddes hmm, yeh the existing terminology is basically wrong though then. the message list is a list of messages, it's not a commitment :S

Reply:
only the hash is actually a commitment

Reply:
so idk what's more confusing, inconsistent but correct naming, or consistent but incorrect naming :D? i'd say the former is better at least for solidity developers to understand the solidity code. i'd go with the former, ie messages/commitment

Reply:
Well, ok, it seems there is lack of consensus on a what the term "commitment" should refer to - the actual value being committed, or some proxy or identifier (the hash) for that thing. IMO both are equally correct in the end so no one's really wrong here 🙂.

Y'all can leave it as messages/commitment, if you prefer though. 👍

Reply:
haha well commitment refers to something that someone has commited to, ie, their is an identifiable individual that has made the commitment. for the hash, we have that, as the hash is actually signed, onchain, by each polkadot validator in the beefy process. for the messages, we don't - nobody really commits to them/signs them directly and they're offchain.

Reply:
I mean it really depends on what we want "commitment" to mean, so I think we are just disputing semantics here.

In my mind, the parachain pledges that a set of messages are fit and proper to be relayed to Ethereum. Those messages are being committed to, ergo they are the commitment. The commitment happens to be uniquely identified by a hash of the messages being committed. In the same way that you can commit code with git and refer to that commit/commitment using a "commit hash" (which is a valid term in git).

I do think `commitment/commitmentHash` is preferable over `message/commitment` as it is more descriptive. It indicates that there is a relationship between the two things.

Reply:
For this PR let's just leave it as it is. I created issue https://github.com/Snowfork/polkadot-ethereum/issues/273 for any further discussions about messages/commitment vs. commitment/commitment hash naming conventions.